sql/sql_acl.cc

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2000, 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
A merge conflict in `sql/sql_acl.cc` inside `replace_user_table()` is not resolved properly.
```
  if (acl_is_utility_user(combo->user.str, combo->host.str, NULL))
  {
    my_error(ER_NONEXISTING_GRANT, MYF(0), combo->user.str, combo->host.str);
    goto end;
  }

  if (!table->key_info)
  {
    my_error(ER_TABLE_CORRUPT, MYF(0), table->s->db.str,
             table->s->table_name.str);
    goto end;
  }
```
should become just
```
  if (acl_is_utility_user(combo->user.str, combo->host.str, NULL))
  {
    my_error(ER_NONEXISTING_GRANT, MYF(0), combo->user.str, combo->host.str);
    goto end;
  }
```

Reply:
fixed